### PR TITLE
Updates for troubleshooting Zap and more clarity on updating the POAM

### PR DESCRIPTION
**Pull request by rcgottlieb**
## Changes proposed in this pull request:
-
-
-

## security considerations
There are no security considerations as this is a documentation fix not code nor configuration.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/run_conmon.md [hunk 1]
```diff
@@ -86,6 +86,44 @@ The following steps are for the `external` scan (except as noted):
...
+
+Change the following level's to debug so the entries look like this:
+```
+logger.paros.name = org.parosproxy.paros 
+logger.paros.level = debug
+
+logger.zap.name = org.zaproxy.zap 
+logger.zap.level = debug
+
+Open Zap, follow the above and open Firefox. Try to go to the server that failed previously.
```
Comment (pburkholder) [inline .github/ISSUE_TEMPLATE/run_conmon.md:105]: Look like ``` should be a couple of lines up.

**Comment by pburkholder** [comment]
Where did you find the hint to try to log4j2 properties?

Also, are you using a Weekly ZAP or the Stable version?

**Comment by rcgottlieb** [comment]
> Where did you find the hint to try to log4j2 properties?
Chris figured it out as he had the same exact issue with the same version of Zap (the latest). Originally we were going to debug this but it started working as soon as we changed those levels. So we changed it back and it worked. Not sure why, but it's a hint to help someone else who might run into the same issue.

> Also, are you using a Weekly ZAP or the Stable version?
Whatever version brew installs is the version I used. It was most likely Stable as I think you have to tell it if you want a weekly version.

File: .github/ISSUE_TEMPLATE/run_conmon.md (hunk 1 above)
Comment (rcgottlieb) [reply .github/ISSUE_TEMPLATE/run_conmon.md]: agreed and done

**Review by pburkholder — APPROVED** [review]

**Comment by pburkholder** [comment]
Thanks for the helpful notes

**Pull request merged**